spec/xtrace.test.sh

```diff
@@ -367,3 +367,33 @@ ok
 [last=0] 'false'
 [last=1] echo ok
 ## END
+
+
+#### Regression: declare -a a+=(v)
+case $SH in dash|mksh) exit ;; esac
+
+a=(1)
+set -x
+declare a+=(2)
+## STDERR:
++ declare a+=(2)
+## END
+## OK bash STDERR:
++ a+=('2')
++ declare a
+## END
+## N-I dash/mksh STDERR:
+## END
+
+
+#### Regression: declare -a a+=(v)
```

Comment:
shouldn't this be named without `declare` ?     They are 2 different cases

Reply:
Ah, right! I fixed it.